src/align.py

```diff
@@ -17,7 +17,21 @@ def get_edits(p: str, q: str) -> tuple[str, str, str]:
     """
     assert len(p) == len(q)
     # FIXME: do the actual calculations here
-    return '', '', ''
+    orig, trans, cigar = "","",""
+
+    for i in range(len(p)):
+        if p[i] != "-" and q[i] != "-":
+            orig += f"{p[i]}"
+            trans += f"{q[i]}"
+            cigar += f"M"
```

Comment:
If `x` is a string, then `f"{x}"` is equivalent to `x`. Since there is no formatting, the string `f"M"` is just `"M"`.